src/ort_genai_c.cpp

```diff
@@ -105,10 +105,24 @@ OgaResult* OGA_API_CALL OgaCreateGeneratorParams(const OgaModel* model, OgaGener
 }
 
 OgaResult* OGA_API_CALL OgaGeneratorParamsSetMaxLength(OgaGeneratorParams* params, size_t max_length) {
```

Comment:
remove this function

Reply:
It's already removed. You might be looking at before Baiju's change

Reply:
Done